pkg/api/norman/customization/aks/listers.go

```diff
@@ -0,0 +1,192 @@
+package aks
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"regexp"
+	"sort"
+
+	"github.com/Azure/azure-sdk-for-go/services/containerservice/mgmt/2020-09-01/containerservice"
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2020-07-01/network"
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/adal"
+	"github.com/mcuadros/go-version"
+)
+
+type virtualNetworksResponseBody struct {
+	Name          string   `json:"name"`
+	ResourceGroup string   `json:"resourceGroup"`
+	Subnets       []subnet `json:"subnets"`
+}
+
+type subnet struct {
+	Name         string `json:"name"`
+	AddressRange string `json:"addressRange"`
+}
+
+var matchAzureNetwork = regexp.MustCompile("/resourceGroups/(.+?)/")
+
+func NewClientAuthorizer(cap *Capabilities) (autorest.Authorizer, error) {
+	oauthConfig, err := adal.NewOAuthConfig(cap.AuthBaseURL, cap.TenantID)
```

Comment:
same here, thought we got rid of tenant

Reply:
We can't get rid of TenantID Azure API calls require it